### ISSUE TITLE
universal is too sensitive

### ISSUE THREAD

**Issue by PalleZingmark** [rossPatton/stylint#32]
This throws a warning:
```css
fubarCalcFunction( $width, $height ) {
  return ( $width*$height );
}
```
This does not:
```css
fubarCalcFunction( $width, $height ) {
  return ( $width * $height );
}
```

**Comment by rossPatton** [comment]
should be less sensitive now. it should only trigger warnings when * is used as a property, not as a value
